catalogd/cmd/catalogd/main.go

```diff
@@ -115,7 +115,7 @@ func main() {
 	flag.StringVar(&certFile, "tls-cert", "", "The certificate file used for serving catalog and metrics. Required to enable the metrics server. Requires tls-key.")
 	flag.StringVar(&keyFile, "tls-key", "", "The key file used for serving catalog contents and metrics. Required to enable the metrics server. Requires tls-cert.")
 	flag.IntVar(&webhookPort, "webhook-server-port", 9443, "The port that the mutating webhook server serves at.")
-	flag.StringVar(&caCertDir, "ca-certs-dir", "", "The directory of CA certificate to use for verifying HTTPS connections to image registries.")
+	flag.StringVar(&pullCertDir, "pull-certs-dir", "", "The directory of CA certificate to use for verifying HTTPS connections to image registries.")
```

Comment:
should we not change it in upstream to avoid conflicts here?

Reply:
I could put back `--ca-certs-dir` and indicate that it is deprecated, but not sure we really need to do that.

Reply:
Why do we need to change the name?
They are still doing the same right

IF we want to change the name why not change in upstream?

Reply:
This _is_ upstream.

Reply:
They are not doing the same thing. We were using the same sets of CAs for multiple _different_ operations (communicating with catalogd and pulling images), and we need those to be able to be configured separately and differently.